updatecli/updatecli.d/plugin-health-scoring.yaml

```diff
@@ -47,6 +47,10 @@ actions:
     scmid: default
     title: Bump `plugin-health-scoring` docker image version to {{ source "latestRelease" }}
     spec:
+      automerged: true
```

Comment:
Ping @alecharp : I realize I approved too quickly: this parameter should be `automerge` as per https://www.updatecli.io/docs/plugins/actions/github/#_parameters

Reply:
I knew I couldn't have made it correctly on the first try..